Computer.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include "LeakedObjectDetector.h"
+
+struct Computer
+{
+    Computer();
+    ~Computer();
+    int numCPUCores = 6;
+    float CPUFrequencyGHz = 3.4f;
+    int memoryMB = 32;
+    int powerNeededW = 430;
+    std::string operatingSystem = "Solaris";
+
+    struct Drive;
+
+    Drive* cDrive;
```

Comment:
Even though you know how to use pointers and allocate on the heap, you should always try to create all of your objects as stack variables, unless you absolutely HAVE to use the heap.
Declare this as a regular `Drive drive`; 
- [ ] revise